### DISCUSSION TITLE
Rancher compatibility check

### DISCUSSION THREAD

**Discussion by aDingil** [kube-hetzner/terraform-hcloud-kube-hetzner#788] [Ideas]
So currently rancher doesnt support kubernetes 1.26.4, but inside the tf. file you can specify to install the latest version of kubernetes and rancher.
But dont know how compatible these settings are.

**Comment by mysticaltech** [comment]
@aDingil You can set `initial_k3s_channel` as explained in the rancher section comments in the kube.tf, set it to a prior version of v1.25 for compatibility.

**Reply by mysticaltech** [reply]
There is no way to run compatibility checks, you have to follow the docs on how to check this, rancher has a compatibility with kube versions on their website.
